src/uct/ib/dc/dc_mlx5.h

```diff
@@ -36,7 +36,10 @@ struct ibv_ravh {
 #  define UCT_DC_RNDV_HDR_LEN   0
 #endif
 
-#define UCT_DC_MLX5_IFACE_MAX_DCIS   16
+#define UCT_DC_MLX5_IFACE_MAX_USER_DCIS 15
+#define UCT_DC_KEEPALIVE_MAX_DCIS       1
```

Comment:
maybe `UCT_DC_KEEPALIVE_NUM_DCIS`, as it is an exact number which is not supposed to vary ?

Reply:
ok, renamed to UCT_DC_KEEPALIVE_NUM_DCIS